<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreCagnotte</class>
 <widget class="QWidget" name="FenetreCagnotte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica Neue</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#logo{ 
 border-width: 2px;
    border-style: none;
    border-radius: 3px;
	border-color: transparent;
background: qlineargradient(spread:pad, x1:1 y1:1, x2:0, y2:0, stop:0 #CFCFCF , stop:1 #F2F2F2);
}

QPushButton{ 
border-width: 2px;
    border-radius: 3px;
border-style: solid;
    border-color: black;
background: qlineargradient(spread:pad, x1:1 y1:1, x2:0, y2:0, stop:0 #CFCFCF , stop:1 #F2F2F2);
}

QTableView {
	background: qlineargradient(spread:pad, x1:0 y1:1, x2:0, y2:0, stop:0 rgba(202, 207, 210,255), stop:1 rgba(242, 243, 244, 255));
	border-color:black; 
	border-style:solid; 
	border-width:2px;
}
QTableView::item{
    height: 31px;

}</string>
  </property>
  <widget class="QPushButton" name="logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Retourner au menu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background: transparent;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/images/logo.png</normaloff>:/new/images/logo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CAGNOTTE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="accepterRetrait">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Accepter une demande de retrait</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ajouterArgentButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ajouter de l'argent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retirerArgentButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>260</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Retirer de l'argent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="consulterContributions">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Consulter mes contributions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cloturerButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Supprimer définitivement la cagnotte ! &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#cloturerButton{ 
border-width: 2px;
    border-radius: 3px;
border-style: solid;
    border-color: black;
background: qlineargradient(spread:pad, x1:1 y1:1, x2:0, y2:0, stop:0 #D98880 , stop:1 #A93226);
}</string>
   </property>
   <property name="text">
    <string>Supprimer la cagnotte</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inviterCagnotte">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{  border-width: 2px;
    border-style: solid;
    border-radius: 3px;
	border-color: black;}</string>
   </property>
   <property name="text">
    <string>Inviter</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{ border: 1px solid black;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Objectif : </string>
    </property>
   </widget>
   <widget class="QLabel" name="objectifValue">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0.00€</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Montant : </string>
    </property>
   </widget>
   <widget class="QLabel" name="montantValue">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>79</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.00€</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_4{
background-color:transparent; 
color:green;}</string>
   </property>
   <property name="text">
    <string>Objectif atteint ! </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>logo</zorder>
  <zorder>label</zorder>
  <zorder>accepterRetrait</zorder>
  <zorder>ajouterArgentButton</zorder>
  <zorder>retirerArgentButton</zorder>
  <zorder>consulterContributions</zorder>
  <zorder>cloturerButton</zorder>
  <zorder>inviterCagnotte</zorder>
  <zorder>label_4</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
